package.json

```diff
@@ -76,7 +76,7 @@
 		"authly": "^3.0.7",
 		"cryptly": "^3.0.8",
 		"flagly": "^0.1.13",
-		"isly": "^0.1.13",
+		"isly": "^0.1.16",
```

Comment:
updated to the latest version

Reply:
updating caused all sorts of issues so we are not upgrading during this PR